napari/_qt/menus/file_menu.py

```diff
@@ -211,6 +228,20 @@ def _add_sample(*args, plg=plugin_name, smp=samp_name):
                 menu.addAction(action)
                 action.triggered.connect(_add_sample)
 
+    def _open_files_w_plugin(self):
+        """Helper method for forcing plugin choice"""
+        self._win._qt_viewer._open_files_dialog(choose_plugin=True)
+
+    def _open_files_as_stack_w_plugin(self):
+        """Helper method for forcing plugin choice"""
+        self._win._qt_viewer._open_files_dialog_as_stack_dialog(
+            choose_plugin=True
+        )
+
+    def _open_folder_w_plugin(self):
+        """Helper method for forcing plugin choice"""
+        self._win._qt_viewer._open_folder_dialog(choose_plugin=True)
```

Comment:
I would love to see some tests that will test these functions (With mocking dialog `show`/`exec`) te be sure that the menu entry will work. Maybe by calling menu items directly.